### ISSUE TITLE
verifying github.com/libdns/gandi@v1.0.2: checksum mismatch

### ISSUE THREAD

**Issue by sewnie** [foxcpp/maddy#555]
# Describe the bug

can't build maddy

# Steps to reproduce
```
git clone https://github.com/foxcpp/maddy.git
cd maddy
./build.sh
```

# Log files

```
-- Version: 0.6.2+e1b3391
-- Building main server executable...
go: downloading github.com/caddyserver/certmagic v0.16.1
go: downloading github.com/urfave/cli/v2 v2.10.2
go: downloading github.com/foxcpp/go-imap v1.0.0-beta.1.0.20220623182312-df940c324887
go: downloading github.com/foxcpp/go-imap-sql v0.5.1-0.20220627220518-df3b66a5b04f
go: downloading golang.org/x/crypto v0.0.0-20220622213112-05595931fe9d
go: downloading github.com/klauspost/cpuid/v2 v2.0.14
go: downloading github.com/libdns/libdns v0.2.1
go: downloading github.com/mholt/acmez v1.0.2
go: downloading github.com/miekg/dns v1.1.50
go: downloading go.uber.org/zap v1.21.0
go: downloading golang.org/x/net v0.0.0-20220622184535-263ec571b305
go: downloading github.com/emersion/go-message v0.16.0
go: downloading github.com/emersion/go-msgauth v0.6.6
go: downloading github.com/emersion/go-smtp v0.15.1-0.20220119142625-1c322d2783aa
go: downloading github.com/emersion/go-sasl v0.0.0-20211008083017-0b9dcfb154ac
go: downloading github.com/foxcpp/go-dovecot-sasl v0.0.0-20200522223722-c4699d7a24bf
go: downloading github.com/go-ldap/ldap/v3 v3.4.3
go: downloading golang.org/x/text v0.3.7
go: downloading github.com/GehirnInc/crypt v0.0.0-20200316065508-bb7000b8a962
go: downloading golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
go: downloading github.com/emersion/go-milter v0.3.3
go: downloading blitiri.com.ar/go/spf v1.5.1
go: downloading github.com/emersion/go-imap-compress v0.0.0-20201103190257-14809af1d1b9
go: downloading github.com/emersion/go-imap-sortthread v1.2.0
go: downloading github.com/foxcpp/go-imap-i18nlevel v0.0.0-20200208001533-d6ec88553005
go: downloading github.com/foxcpp/go-imap-namespace v0.0.0-20200802091432-08496dd8e0ed
go: downloading github.com/prometheus/client_golang v1.12.2
go: downloading github.com/google/uuid v1.3.0
go: downloading github.com/libdns/cloudflare v0.1.0
go: downloading github.com/libdns/digitalocean v0.0.0-20220518195853-a541bc8aa80f
go: downloading github.com/libdns/gandi v1.0.2
verifying github.com/libdns/gandi@v1.0.2: checksum mismatch
        downloaded: h1:6Z94oPHjL2H+vQoNU4CwZP7C5W0vFnSEyyk+NNyK1Z4=
        go.sum:     h1:1Ts8UpI1x5PVKpOjKC7Dn4+EObndz9gm6vdZnloHSKQ=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

# Environment information
* go version: 1.19.3

**Comment by sewnie** [comment]
solved temporariliy with `export GOPROXY="https://proxy.golang.org,direct"`

**Comment by foxcpp** [comment]
Closed as duplicate of #461. Lets continue the discussion there.

**Comment by sewnie** [comment]
i know it was a duplicate; i opened this issue on purpose to grant it some attention.
